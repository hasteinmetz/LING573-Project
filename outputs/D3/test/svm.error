Traceback (most recent call last):
  File "/projects/assigned/2122_ling573_elibales/repo/src/lexical_features.py", line 84, in <module>
    svm.fit(vectors, labels)
NameError: name 'vectors' is not defined
