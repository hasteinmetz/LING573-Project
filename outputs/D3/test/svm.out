reading sentences...
create lexical vector...
getting tf-idf...
normalizing vectors...
[[0.         0.         0.         ... 0.         0.         0.        ]
 [0.         0.         0.         ... 0.         0.         0.        ]
 [0.         0.         0.         ... 0.         0.         0.        ]
 ...
 [0.91766294 0.         0.         ... 0.         0.         0.        ]
 [0.         0.         0.         ... 0.         0.         0.        ]
 [0.         0.         0.         ... 0.         0.         0.        ]]
